bazel/envoy_build_system.bzl

```diff
@@ -43,7 +43,27 @@ def envoy_linkopts():
             # See note here: http://luajit.org/install.html
             "-pagezero_size 10000", "-image_base 100000000",
         ],
+        "//bazel:enable_exported_symbols": [
```

Comment:
I think this could be a separate select statement which is `+` with the existing one. You can actually factor out a bunch of common list entries into a separate item that is `+` with this as well.